phpBB/phpbb/error_collector.php

```diff
@@ -16,15 +16,28 @@
 class error_collector
 {
 	var $errors;
+	var $error_types;
 
-	function __construct()
+	/**
+	 * Constructor.
+	 *
+	 * The variable $error_types may be set to a mask of PHP error types that
+	 * the collector should keep, e.g. `E_ALL`. If unset, the current value of
+	 * the error_reporting() function will be used to determine which errors
+	 * the collector will keep.
+	 *
+	 * @see PHPBB3-13306
```

Comment:
The tracker page has a longer explanation of the reason for this, I see other projects do it that way and didn't know if there was a style convention for this.